web-frontend/src/pages/TransferCoins/SendCoins/ContactList.tsx

```diff
@@ -7,16 +7,16 @@ import {
   MassaLogo,
 } from '@massalabs/react-ui-kit';
 import { formatStandard } from '../../../utils/massaFormat';
-import { useResource } from '../../../custom/api';
 import { AccountObject } from '../../../models/AccountModel';
 
-function AccountSelect({ ...props }) {
-  const { onClose, setRecipient, account: currentAccount } = props;
+export interface ContactListProps {
```

Comment:
```suggestion
interface ContactListProps {
```

**TIPS:** if you don't use this interface outside this file, remove `export` flag 👍🏻

Reply:
good catch :) I was using it but I moved it